spec/js/ko-components/templates/orifice-calculator-spec.coffee

```diff
@@ -0,0 +1,38 @@
+describe 'template-orifice-calculator-spec', ->
```

Comment:
This file should not be in templates. Templates are just what the name says a template and generally there is very little logic to test on them.

I think this should live under spec/js/ko-components/views/orifice-calculator-spec.coffee